GOVERNANCE.md

```diff
@@ -72,8 +72,10 @@ membership beyond these rules.
 
 The CTC may add additional members to the CTC by a standard CTC motion.
 
-A CTC member may be removed from the CTC by voluntary resignation, or by
-a standard CTC motion.
+When a CTC member's participation in CTC matters has become minimal for a
```

Comment:
> Writing down a shared understanding of what precisely the role of a CTC member is seems like it would only help everyone understand their responsibilities. Not doing so leaves a lot of room for interpretation, and thus confusion.

@misterdjules You raise a valid and important point. While I *think* my categorization is shared by most others on the CTC, I don't actually know that, so let's find out: https://github.com/nodejs/node/pull/7744